src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -709,6 +706,10 @@ Disabling this module or closing PowerToys will unmute the microphone and camera
     <value>Fill</value>
     <comment>Refers to filling an image into a certain size. It could overflow</comment>

```

Comment:
No period at the end? (3 times)

Reply:
It's a comment in the resource file - so not nothing that is user facing.. and not touched by this PR.